src/main/resources/postfixTemplates/DbgrPostfixTemplate/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+Creates <code>dbg!</code> macro invocation from expression.
```

Comment:
Let's rewrite description. Currently, it's exact copy of `dbg` postfix template description and it doesn't describes what `dbgr` does

Reply:
What would you suggest to write here? 🤔

Reply:
For example, `Creates <code>dbg!</code> macro invocation from expression reference.` or something limilar

Reply:
Thank you! Done.